### DISCUSSION TITLE
Native Database Management

### DISCUSSION THREAD

**Discussion by TheoCouss** [caprover/caprover#1546] [Ideas]
Hello!

I've been using Caprover for a year now
During this year I faced some issues with databases (data loss, permissions issues, user auth mode etc..) that gave me this idea; 
What If we were able to install, manage, backup and replicate databases the same way we use managed databases from AWS, DO or OVH etc.. ?
 
It still uses containers as usual but with a UI native to Caprover to manage them. 
(Being able to create users, manage simple permissions, create database a new database etc.. )

This can make Caprover way more easy to use and make cleaner one-click install files!

**Comment by githubsaturn** [comment]
That on itself is a separate project. There will no "generic" database management ever. Every single database has their own way of managing the database. 

I personally would love to see that application and I'll use it. But I don't see that it being a built-in component in CapRover as, like I said, a very curated database management system.

**Reply by TheoCouss** [reply]
I completely agree with your point of view; It would take too much effort to support the compatibility of all the databases. And even if it was implemented, at the begining there would be a lack of features that could make it unusable or bypassed by pma or other database management software, ultimately it's not worth it.

I have a lot of other ideas that can make this software easier to use, I'll focus on that
